js/client.js

```diff
@@ -1722,18 +1723,25 @@ function toId() {
 			if (room.requestLeave && !room.requestLeave(e)) return false;
 			return this.removeRoom(id);
 		},
-		renameRoom: function (id, newid) {
+		renameRoom: function (id, newtitle, newid) {
```

Comment:
```suggestion
		renameRoom: function (id, newid, newtitle) {
```

Bikeshedding, but `newid` should be first for backwards compatibility reasons.